src/main/schemas/territories.schema.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "$id": "http://isdcf.com/ns/json-schemas/registries/territories/1.0.0-beta.1",
+  "$comment": "Copyright, ISDCF <info@isdcf.com>",
+  "title": "Schema for the Territories Registry of the ISDCF",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "properties": {
+      "comments": {
+        "type": "array",
+        "items": {
+          "type": "string"
+        }
+      },
+      "name": {
```

Comment:
Update field name to "dcncTerritory", so that it's similar to the language JSON and clear that it's the DCNC name, and we can add Display name on the HTML like languages using CLDR.

Reply:
Since there are no universal/accepted name for territories, in contrast to language names defined by CLDR, the name assigned by ISDCF is the de-facto name, so no need to qualify the name with _dcnc_/

Reply:
ISO 3166-1 published in English and French, with the names normalized in both uppercase and lower case. I pulled those in the past and converted to Camel case for use on https://www.smptedcp.com/status. I know that the naming for DCNC doesn't always match, case in point "United Kingdom of Great Britain and Northern Ireland"

Reply:
ISO 3166-1 does not include _Dubai_. Until/unless there is a comprehensive mapping between region code and region name, I suggest ISDCF makes up it own.

Reply:
It's in ISO 3166-2:AE. If we cannot use any source for automagically grabbing, then I suggest we conform to the 3166 names. Happy to highlight which ones don't match.

Reply:
@SteveLLamb I like the idea of rationalizing the names. What about we do this after merging in the current names?

Reply:
As discussed we can conform a future name field to a standard later (be it 3166, IANA or M.49) and add fields as needed. To anticipate that being done, I will think the original field should be dcncTerritory so it doesn't have to be renamed as such later, as we had to do with languages.

Reply:
Implemented at 36a00619ed228f7ef4732c7629561162990ced5a

Reply:
Great, thanks!